MergerCli/Program.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using MergerCli.Utils;
```

Comment:
Add log indicating a resume, and some info on the resume operation.

Reply:
added